content/content-strategy/define-user-content-needs/create-actionable-artefacts/methods.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: text/textblock
+---
+
+Choose a method and outcome that will help you to focus on the tasks people are trying to complete. Your skill will be to know where and when to apply the right method to the right context. 
+
+Every project and stakeholder group is different and you will need to achieve different outcomes and agendas. Think about the intent and the outcome to find the right set of methods for that context. 
+
+### [1]Journey maps
```

Comment:
Incorrect display level. Needs to be `### [2]` to show a heading 3. Otherwise `####` for a heading 4.

Reply:
Steve fixed